packages/app/src/systems/Welcome/machines/welcomeMachine.ts

```diff
@@ -411,8 +411,8 @@ export const welcomeMachine = createMachine(
           value: wallet.address.toHexString(),
         };
 
-        const { value: hasMint1 } = await token1.functions.has_mint(addressId).get();
-        const { value: hasMint2 } = await token2.functions.has_mint(addressId).get();
+        const { value: hasMint1 } = await token1.functions.has_mint(addressId).call();
```

Comment:
`get` methods should be change to `dryRun` as this methods are made to be readonly.